modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/test/OpenAPIV3ParserTest.java

```diff
@@ -2296,10 +2291,6 @@ private OpenAPI doRelativeFileTest(String location) {
         assertEquals(refInDefinitions.getDescription(), "The example model");
         expectedPropertiesInModel(refInDefinitions, "foo", "bar");
 
```

Comment:
It's deleted cause it's not correct anymore. We shouldn't get array where we parsed schema with object.